music_assistant/providers/sonos_s1/__init__.py

```diff
@@ -362,6 +377,25 @@ async def poll_player(self, player_id: str) -> None:
 
     async def discover_players(self) -> None:
         """Discover Sonos players on the network."""
+        manual_ip_config: str | None
+        if (manual_ip_config := self.config.get_value(CONF_IPS)) is not None:  # type: ignore[assignment]
```

Comment:
let's not completely ignore typing here but help the type checker a bit by casting it.
Maybe even defining it above is enough:

manual_ip_config: str